sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -408,3 +408,13 @@ case class DescribeFunction(
     }
   }
 }
+
+case class SetDatabaseCommand(databaseName: String) extends RunnableCommand {
```

Comment:
If you mean should we move parsing `TOK_SWITCHDATABASE` to CatalystQl, as currently I see SQLContext use SparkQl not CatalystQl, this should not be a problem now. I have no strong opinion about this. We can move it now or when Catalyst supports database. How do you think?